components/SerialServer/src/serial.c

```diff
@@ -387,12 +387,23 @@ static void handle_char(uint8_t c)
             last_out = -1;
             statemachine = 1;
             break;
+        case 's':
+            printf("%s", COLOR_RESET);
```

Comment:
I think this also needs `fflush(stdout);` because the lack of \n will cause the reset code to be buffered and only print after the `seL4_DebugDumpScheduler();`

Reply:
(Or emitting a `\n` after the COLOR_RESET would work and would ensure that the scheduler dump started on a new line too).

Reply:
Added the newline.